src/phoenix/config.py

```diff
@@ -113,3 +117,7 @@ def get_env_host() -> str:
 
 def get_env_collector_endpoint() -> Optional[str]:
     return os.getenv(ENV_PHOENIX_COLLECTOR_ENDPOINT)
+
+
+def get_env_project_name() -> str:
+    return os.getenv(ENV_PHOENIX_PROJECT_NAME) or "config"
```

Comment:
"config" seems a strange default project name. Why not something like "default"?

Reply:
Oh damn must have been copilot 🤦🏻